### PR TITLE
New Single Cell Tutorial in R

### PR DESCRIPTION
**Pull request by Camila-goclowski**
Built Filter, Plot, and Explore tutorial in RMarkdown using Seurat in R. First contribution to the GTN.

### CODE REVIEW

**Comment by pavanvidem** [comment]
wow! the first Seurat-based tutorial on GTN. Thanks! I am happy to review it once you're done with the draft.

**Comment by hexylena** [comment]
Hi @Camila-goclowski not sure how I missed this one. Very exciting, I will make some formatting comment.s

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.html
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.html:1]: Please remove this file, it seems to be a pandoc rendered copy of the markdown which isn't needed in our case :)

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md:10]:
```suggestion
Let's get our dataset loaded into Galaxy, first. 
```

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md:13]:
```suggestion
> Run {% tool [EBI SCXA Data Retrieval](toolshed.g2.bx.psu.edu/repos/ebi-gxa/retrieve_scxa/retrieve_scxa/v0.0.2+galaxy2) %} with the following parameters: 
```

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md:22]:
```suggestion
## Open RStudio in Galaxy 

You should now see a matrix.mtx, genes.tsv, barcodes.tsv, and exp_design.tsv files in your Galaxy history. For the rest of the workflow, let's move onto RStudio and get coding!

> <hands-on-title>Open RStudio in Galaxy</hands-on-title>
> Run {% tool [RStudio](interactive_tool_rstudio)}
{: .hands_on}
```

Comment (hexylena) [reply topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md:22]: adds more spacing for better legibility, but the important change is only the `> Run` before the tool link.

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md:69]:
```suggestion
So, for example, our matrix was the first to be saved in our history. As such, we will ask for the filepath for the first slot in our history with the following line:  
```

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md
Comment (hexylena) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md:175]:
```suggestion

Ideally, we would like to see a relatively even distribution of counts for each individual (or batch) but if there isn’t, fear not, we can regress this variable out in a later step.
```

Comment (hexylena) [reply topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md:175]: you have a number of these, do you intend for them to be rendered in some special way? Should it be a comment box maybe?

Otherwise I think it makes sense to just remove the `>` that starts an unstyled (in the GTN framework) blockquote. By my reading, in most cases in this tutorial, it would be fine without those blockquotes.

**Review by hexylena — COMMENTED** [review]
Some small formatting comments but amazing, another R tutorial!!

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.html
Comment (Camila-goclowski) [reply topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.html:1]: Thank you for your comments. I'm still learning the markdown formatting and these are incredibly helpful!

**Comment by shiltemann** [comment]
nice! don't forget to add yourself to our CONTRIBUTORS.yaml file :) (should also fix the linting error)

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md
Comment (mtekman) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md:177]: maybe mention that uppercase "MT" might be required in some datasets

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md
Comment (pavanvidem) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/tutorial.md:604]: I got a different plot
image

**Review by pavanvidem — COMMENTED** [review]
@Camila-goclowski wonderful! very detailed. Thanks!!

**Comment by pavanvidem** [comment]
Maybe also please end the tutorial with showing how to save the object and some plots to the history.

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md
Comment (shiltemann) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md:18]:
```suggestion
We’ll provided you with experimental data to analyse from a mouse dataset of fetal growth restriction [Bacon et al. 2018]({% cite Bacon2018 %}). This is the full dataset generated from [this tutorial]({% link topics/single-cell/tutorials/scrna-case_alevin/tutorial.html %}). 
```

please use the `{% link topics/../../ %}` syntax for links to training materials.

And if you want to link to a reference/citation, please include it in this tutorial (so add a `tutorial.bib` file and put the citation in there (in this case you can copy it from the other tutorial's bib file)

File: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md
Comment (shiltemann) [inline topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md:6]: this is unrelated to the linting error, but I think you want `.html` here
```suggestion
> This tutorial is significantly based on the Seurat documentation({% cite Satija2015 %}) as well as [Seurat's Guided Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial.html).
```

**Review by hexylena — APPROVED** [review]
I'm going to merge this and fix the build if it fails when I can

**Comment by hexylena** [comment]
Ok, opened a new PR based off of `main` after this was merged and, things work fine. I suspect it was a branch specific issue, it might've been solved by updating this PR to be rebased against the latest main branch but that's also might not have been worth it.

So, yay, PR merged, congrats @Camila-goclowski on your first tutorial! Welcome to the GTN

**Pull request merged**
